puppet/modules/consul/spec/classes/install_spec.rb

```diff
@@ -56,13 +56,13 @@
       should contain_file('/opt/consul-backinator-1.3').with(
         :ensure => 'directory',
       )
-      should contain_file('/opt/consul-backinator-1.3/consul-backinator').with(
+      should contain_file('/opt/consul-backinator-1.3/consul-backinator-1.3').with(
```

Comment:
Where is that change coming from?

Reply:
@simonswine The binary is named `consul-backinator-1.3` in the tar ball. When we had the `creates` before in the puppet resource it would create that file but would be empty.

I'm unsure the right was to change it to just `consul-backinator` if that's what we what to do

Reply:
This is quite a lot of change and I am not too sure how extract: false would have ever worked.

I would feel way better if we would rename the whole thing instead of having the version in there again.

Also happy to upgrade that component, if we touch it already

Reply:
I agree about it not working. Happy to have another look and point it to a later version